### DISCUSSION TITLE
Generic filters works differently than in uBO.

### DISCUSSION THREAD

**Discussion by dportvine** [uBlockOrigin/uBOL-home#226] [General]
Can this be fixed on MV3?

`https://forum.overclockers.ua/viewtopic.php?f=23&t=283043&start=40`

<details>
<summary>Screen</summary>

https://github.com/user-attachments/assets/92e3767b-4141-432a-955c-3d5326c8c970
</details>

**Comment by gorhill** [comment]
uBO would exihibit the same issue if it was not caching the cosmetic selectors found by the surveyor. Such caching is not possible with uBO Lite because of its declarative approach.

I have hope that if and once [this webextensions issue](https://github.com/w3c/webextensions/issues/403) is resolved, the CSS injection would happen in a more timely manner.

**Comment by gorhill** [comment]
The following filter could be added to RUAdList:

    overclockers.ua##[class^="ad_slot"]

**Reply by dportvine** [reply]
@dimisa-RUAdList

Can you add this rule for uBOL?

**Reply by dimisa-RUAdList** [reply]
Ok: https://github.com/easylist/ruadlist/commit/8d02616bc218e984b6cf8385843370927b67c5f8
